object_storage_api/stores/image.py

```diff
@@ -37,3 +38,22 @@ def upload(self, image_id: str, image_metadata: ImagePostMetadataSchema, upload_
         )
 
         return object_key
+
+    def create_presigned_url(self, object_key: str, file_name: str) -> HttpUrl:
```

Comment:
```suggestion
    def create_presigned_get(self, object_key: str, file_name: str) -> HttpUrl:
```
Would match with attachments `create_presigned_post` as this is a get presigned URL.